.eslintrc.js

```diff
@@ -58,5 +58,6 @@ module.exports = {
   rules: {
     'react/prop-types': 'off',
     'no-underscore-dangle': 'off',
+    'import/prefer-default-export': 'off',
```

Comment:
Nervige Linter-Rule deaktiviert, um die Existenz von "default-exports" in einem Module nicht zu erzwingen.